src/openfermion/utils/_lcu_util.py

```diff
@@ -17,6 +17,24 @@
 import math
 
 
+def lambda_norm(diagonal_operator):
```

Comment:
This name isn't very descriptive, something like 'triangle_inequality_norm_bound' would be clearer.